diff --git a/node_modules/@edx/frontend-component-footer/dist/components/Footer.js b/node_modules/@edx/frontend-component-footer/dist/components/Footer.js
index 7bed8d7..5c99d7d 100644
--- a/node_modules/@edx/frontend-component-footer/dist/components/Footer.js
+++ b/node_modules/@edx/frontend-component-footer/dist/components/Footer.js
@@ -1,16 +1,16 @@
 function _typeof(o) { "@babel/helpers - typeof"; return _typeof = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function (o) { return typeof o; } : function (o) { return o && "function" == typeof Symbol && o.constructor === Symbol && o !== Symbol.prototype ? "symbol" : typeof o; }, _typeof(o); }
-function _classCallCheck(a, n) { if (!(a instanceof n)) throw new TypeError("Cannot call a class as a function"); }
-function _defineProperties(e, r) { for (var t = 0; t < r.length; t++) { var o = r[t]; o.enumerable = o.enumerable || !1, o.configurable = !0, "value" in o && (o.writable = !0), Object.defineProperty(e, _toPropertyKey(o.key), o); } }
-function _createClass(e, r, t) { return r && _defineProperties(e.prototype, r), t && _defineProperties(e, t), Object.defineProperty(e, "prototype", { writable: !1 }), e; }
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, _toPropertyKey(descriptor.key), descriptor); } }
+function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); Object.defineProperty(Constructor, "prototype", { writable: false }); return Constructor; }
 function _toPropertyKey(t) { var i = _toPrimitive(t, "string"); return "symbol" == _typeof(i) ? i : i + ""; }
 function _toPrimitive(t, r) { if ("object" != _typeof(t) || !t) return t; var e = t[Symbol.toPrimitive]; if (void 0 !== e) { var i = e.call(t, r || "default"); if ("object" != _typeof(i)) return i; throw new TypeError("@@toPrimitive must return a primitive value."); } return ("string" === r ? String : Number)(t); }
-function _callSuper(t, o, e) { return o = _getPrototypeOf(o), _possibleConstructorReturn(t, _isNativeReflectConstruct() ? Reflect.construct(o, e || [], _getPrototypeOf(t).constructor) : o.apply(t, e)); }
-function _possibleConstructorReturn(t, e) { if (e && ("object" == _typeof(e) || "function" == typeof e)) return e; if (void 0 !== e) throw new TypeError("Derived constructors may only return object or undefined"); return _assertThisInitialized(t); }
-function _assertThisInitialized(e) { if (void 0 === e) throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); return e; }
+function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function"); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, writable: true, configurable: true } }); Object.defineProperty(subClass, "prototype", { writable: false }); if (superClass) _setPrototypeOf(subClass, superClass); }
+function _setPrototypeOf(o, p) { _setPrototypeOf = Object.setPrototypeOf ? Object.setPrototypeOf.bind() : function _setPrototypeOf(o, p) { o.__proto__ = p; return o; }; return _setPrototypeOf(o, p); }
+function _createSuper(Derived) { var hasNativeReflectConstruct = _isNativeReflectConstruct(); return function _createSuperInternal() { var Super = _getPrototypeOf(Derived), result; if (hasNativeReflectConstruct) { var NewTarget = _getPrototypeOf(this).constructor; result = Reflect.construct(Super, arguments, NewTarget); } else { result = Super.apply(this, arguments); } return _possibleConstructorReturn(this, result); }; }
+function _possibleConstructorReturn(self, call) { if (call && (_typeof(call) === "object" || typeof call === "function")) { return call; } else if (call !== void 0) { throw new TypeError("Derived constructors may only return object or undefined"); } return _assertThisInitialized(self); }
+function _assertThisInitialized(self) { if (self === void 0) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return self; }
 function _isNativeReflectConstruct() { try { var t = !Boolean.prototype.valueOf.call(Reflect.construct(Boolean, [], function () {})); } catch (t) {} return (_isNativeReflectConstruct = function _isNativeReflectConstruct() { return !!t; })(); }
-function _getPrototypeOf(t) { return _getPrototypeOf = Object.setPrototypeOf ? Object.getPrototypeOf.bind() : function (t) { return t.__proto__ || Object.getPrototypeOf(t); }, _getPrototypeOf(t); }
-function _inherits(t, e) { if ("function" != typeof e && null !== e) throw new TypeError("Super expression must either be null or a function"); t.prototype = Object.create(e && e.prototype, { constructor: { value: t, writable: !0, configurable: !0 } }), Object.defineProperty(t, "prototype", { writable: !1 }), e && _setPrototypeOf(t, e); }
-function _setPrototypeOf(t, e) { return _setPrototypeOf = Object.setPrototypeOf ? Object.setPrototypeOf.bind() : function (t, e) { return t.__proto__ = e, t; }, _setPrototypeOf(t, e); }
+function _getPrototypeOf(o) { _getPrototypeOf = Object.setPrototypeOf ? Object.getPrototypeOf.bind() : function _getPrototypeOf(o) { return o.__proto__ || Object.getPrototypeOf(o); }; return _getPrototypeOf(o); }
 import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
@@ -24,15 +24,16 @@ var EVENT_NAMES = {
   FOOTER_LINK: 'edx.bi.footer.link'
 };
 var SiteFooter = /*#__PURE__*/function (_React$Component) {
+  _inherits(SiteFooter, _React$Component);
+  var _super = _createSuper(SiteFooter);
   function SiteFooter(props) {
     var _this;
     _classCallCheck(this, SiteFooter);
-    _this = _callSuper(this, SiteFooter, [props]);
-    _this.externalLinkClickHandler = _this.externalLinkClickHandler.bind(_this);
+    _this = _super.call(this, props);
+    _this.externalLinkClickHandler = _this.externalLinkClickHandler.bind(_assertThisInitialized(_this));
     return _this;
   }
-  _inherits(SiteFooter, _React$Component);
-  return _createClass(SiteFooter, [{
+  _createClass(SiteFooter, [{
     key: "externalLinkClickHandler",
     value: function externalLinkClickHandler(event) {
       var label = event.currentTarget.getAttribute('href');
@@ -53,12 +54,42 @@ var SiteFooter = /*#__PURE__*/function (_React$Component) {
         intl = _this$props.intl;
       var showLanguageSelector = supportedLanguages.length > 0 && onLanguageSelected;
       var config = this.context.config;
+      if (config.CATALOG_BASE_URL) {
+        var tos_url = "".concat(config.CATALOG_BASE_URL, "general-terms-and-conditions/");
+        var privacy_url = "".concat(config.CATALOG_BASE_URL, "privacy-policy/");
+      } else {
+        var tos_url = "https://swissmooc.ch/general-terms-and-conditions/";
+        var privacy_url = "https://swissmooc.ch/privacy-policy/";
+      }
       return /*#__PURE__*/React.createElement("footer", {
         role: "contentinfo",
         className: "footer d-flex border-top py-3 px-4"
       }, /*#__PURE__*/React.createElement("div", {
         className: "container-fluid d-flex"
+      }, /*#__PURE__*/React.createElement("ul", {
+        className: "list-unstyled"
+      }, /*#__PURE__*/React.createElement("li", null, showLanguageSelector && /*#__PURE__*/React.createElement(LanguageSelector, {
+        options: supportedLanguages,
+        onSubmit: onLanguageSelected
+      })), /*#__PURE__*/React.createElement("li", {
+        style: {
+          fontSize: "0.6875em"
+        },
+        className: "pt-1"
+      }, "\xA9 ", config.SITE_NAME, ". All rights reserved except where noted.  edX, Open edX and their respective logos are registered trademarks of edX Inc."), /*#__PURE__*/React.createElement("li", {
+        style: {
+          fontSize: "0.6875em"
+        },
+        className: "pt-1"
       }, /*#__PURE__*/React.createElement("a", {
+        className: "text-primary-500 mr-1",
+        href: tos_url
+      }, intl.formatMessage(messages['footer.catalogLinks.tos'])), /*#__PURE__*/React.createElement("a", {
+        className: "text-primary-500 ml-1",
+        href: privacy_url
+      }, intl.formatMessage(messages['footer.catalogLinks.privacy'])))), /*#__PURE__*/React.createElement("div", {
+        className: "flex-grow-1"
+      }), /*#__PURE__*/React.createElement("a", {
         className: "d-block",
         href: config.LMS_BASE_URL,
         "aria-label": intl.formatMessage(messages['footer.logo.ariaLabel'])
@@ -68,14 +99,10 @@ var SiteFooter = /*#__PURE__*/function (_React$Component) {
         },
         src: logo || config.LOGO_TRADEMARK_URL,
         alt: intl.formatMessage(messages['footer.logo.altText'])
-      })), /*#__PURE__*/React.createElement("div", {
-        className: "flex-grow-1"
-      }), showLanguageSelector && /*#__PURE__*/React.createElement(LanguageSelector, {
-        options: supportedLanguages,
-        onSubmit: onLanguageSelected
-      })));
+      }))));
     }
   }]);
+  return SiteFooter;
 }(React.Component);
 SiteFooter.contextType = AppContext;
 SiteFooter.propTypes = {
diff --git a/node_modules/@edx/frontend-component-footer/dist/components/Footer.messages.js b/node_modules/@edx/frontend-component-footer/dist/components/Footer.messages.js
index da4ff23..80b7353 100644
--- a/node_modules/@edx/frontend-component-footer/dist/components/Footer.messages.js
+++ b/node_modules/@edx/frontend-component-footer/dist/components/Footer.messages.js
@@ -1,5 +1,15 @@
 import { defineMessages } from '@edx/frontend-platform/i18n';
 var messages = defineMessages({
+  'footer.catalogLinks.tos': {
+    id: 'footer.catalogLinks.tos',
+    defaultMessage: 'Terms of Service & Honor Code',
+    description: 'This is screenreader text for the Catalog Terms of Service policy.'
+  },
+  'footer.catalogLinks.privacy': {
+    id: 'footer.catalogLinks.privacy',
+    defaultMessage: 'Privacy Policy',
+    description: 'This is screenreader text for the Catalog Privacy policy URL.'
+  },
   'footer.socialLinks.srText.facebook': {
     id: 'footer.socialLinks.srText.facebook',
     defaultMessage: 'Like edX on Facebook',
diff --git a/node_modules/@edx/frontend-component-footer/dist/components/LanguageSelector.js b/node_modules/@edx/frontend-component-footer/dist/components/LanguageSelector.js
index d4fc522..de7ec65 100644
--- a/node_modules/@edx/frontend-component-footer/dist/components/LanguageSelector.js
+++ b/node_modules/@edx/frontend-component-footer/dist/components/LanguageSelector.js
@@ -1,7 +1,7 @@
 var _excluded = ["intl", "options", "onSubmit"];
-function _extends() { return _extends = Object.assign ? Object.assign.bind() : function (n) { for (var e = 1; e < arguments.length; e++) { var t = arguments[e]; for (var r in t) ({}).hasOwnProperty.call(t, r) && (n[r] = t[r]); } return n; }, _extends.apply(null, arguments); }
-function _objectWithoutProperties(e, t) { if (null == e) return {}; var o, r, i = _objectWithoutPropertiesLoose(e, t); if (Object.getOwnPropertySymbols) { var s = Object.getOwnPropertySymbols(e); for (r = 0; r < s.length; r++) o = s[r], t.includes(o) || {}.propertyIsEnumerable.call(e, o) && (i[o] = e[o]); } return i; }
-function _objectWithoutPropertiesLoose(r, e) { if (null == r) return {}; var t = {}; for (var n in r) if ({}.hasOwnProperty.call(r, n)) { if (e.includes(n)) continue; t[n] = r[n]; } return t; }
+function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
+function _objectWithoutProperties(source, excluded) { if (source == null) return {}; var target = _objectWithoutPropertiesLoose(source, excluded); var key, i; if (Object.getOwnPropertySymbols) { var sourceSymbolKeys = Object.getOwnPropertySymbols(source); for (i = 0; i < sourceSymbolKeys.length; i++) { key = sourceSymbolKeys[i]; if (excluded.indexOf(key) >= 0) continue; if (!Object.prototype.propertyIsEnumerable.call(source, key)) continue; target[key] = source[key]; } } return target; }
+function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }
 import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-platform/i18n';
@@ -20,14 +20,7 @@ var LanguageSelector = function LanguageSelector(_ref) {
     onSubmit: handleSubmit
   }, props), /*#__PURE__*/React.createElement("div", {
     className: "form-group"
-  }, /*#__PURE__*/React.createElement("label", {
-    htmlFor: "site-footer-language-select",
-    className: "d-inline-block m-0"
-  }, /*#__PURE__*/React.createElement(FormattedMessage, {
-    id: "footer.languageForm.select.label",
-    defaultMessage: "Choose Language",
-    description: "The label for the laguage select part of the language selection form."
-  })), /*#__PURE__*/React.createElement("select", {
+  }, /*#__PURE__*/React.createElement("select", {
     id: "site-footer-language-select",
     className: "form-control-sm mx-2",
     name: "site-footer-language-select",
